src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -352,6 +460,7 @@ extern "C" int32_t EnumCalendarInfo(
 		case LongDates:
 			// TODO: need to replace the "EEEE"s with "dddd"s for .net
 			// Also, "LLLL"s to "MMMM"s
+			// Also, "G"s to "g"s
 			return InvokeCallbackForDateTimePattern(locale, "eeeeMMMMddyyyy", callback, context);
```

Comment:
why we hard code the date pattern (i.e. eeeeMMMMddyyyy)? was it suppose we get the default pattern from ICU? this apply to other cases like YearMonth for example.